WordPress/src/main/java/org/wordpress/android/ui/sitecreation/SiteCreationBaseFormFragment.java

```diff
@@ -47,14 +47,15 @@ protected ViewGroup createMainView(LayoutInflater inflater, ViewGroup container,
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         ViewGroup rootView = createMainView(inflater, container, savedInstanceState);
-        setupContent(rootView);
         return rootView;
     }
 
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
+        setupContent((ViewGroup) getView().getRootView());
+
         Toolbar toolbar = view.findViewById(R.id.toolbar_main);
         ((AppCompatActivity) getActivity()).setSupportActionBar(toolbar);
```

Comment:
This is unrelated to this PR, but AS is complaining about a potential NPE here, should we change it to:
```suggestion
        ((AppCompatActivity) requireActivity()).setSupportActionBar(toolbar);
```